test/node/download-lsd-magnet.js

```diff
@@ -0,0 +1,37 @@
+const fixtures = require('webtorrent-fixtures')
+const MemoryChunkStore = require('memory-chunk-store')
+const test = require('tape')
+const WebTorrent = require('../../')
+
+test('Download using LSD (via magnet uri)', function (t) {
+  t.plan(3)
+
+  const client1 = new WebTorrent({ dht: false, tracker: false, lsd: true })
+  const client2 = new WebTorrent({ dht: false, tracker: false, lsd: true })
+
+  client1.on('error', function (err) { t.fail(err) })
+  client1.on('warning', function (err) { t.fail(err) })
+
+  client2.on('error', function (err) { t.fail(err) })
+  client2.on('warning', function (err) { t.fail(err) })
+
+  // Start seeding
+  client2.seed(fixtures.leaves.content, {
+    name: 'Leaves of Grass by Walt Whitman.epub',
+    announce: []
+  }, function (torrent) {
+    torrent.discovery.lsd._announce() // hack to send a lsd announce skipping 5min interval
```

Comment:
You might be able to clean up this hack by creating `client1` before `client2` so that the first announce that `client2` sends here https://github.com/webtorrent/bittorrent-lsd/blob/ded22d3e79d97b5082f5eff667be649113596850/index.js#L141 will immediately notify `client1`.

Reply:
Thank you! That was a helpful tip 😉